pkg/machines/components/storagePools/storagePoolVolumesTab.jsx

```diff
@@ -109,13 +111,17 @@ export class StoragePoolVolumesTab extends React.Component {
                 <InlineNotification type='danger' text={this.state.deleteError}
                     detail={this.state.deleteErrorDetail}
                     onDismiss={() => this.setState({ deleteError: undefined }) } /> }
-                <ListingTable variant='compact'
-                    actions={actions}
-                    aria-label={`Storage pool ${storagePool.name} Volumes`}
-                    emptyCaption={_("No storage volumes defined for this storage pool")}
-                    columns={columnTitles}
-                    onSelect={this.onSelect}
-                    rows={rows} />
+                {storagePool.active
+                    ? <ListingTable variant='compact'
+                        actions={actions}
+                        aria-label={`Storage pool ${storagePool.name} Volumes`}
+                        emptyCaption={_("No storage volumes defined for this storage pool")}
+                        columns={columnTitles}
+                        onSelect={this.onSelect}
+                        rows={rows} />
+                    : <EmptyState>
```

Comment:
How about integrating this to the emptyCaption? 
```
emptyCaption={storagePool.active ?_("No storage volumes defined for this storage pool") : _("Activate the storage pool to administer volumes")}
```

Reply:
That would leave the table header with "Name", "UsedBy", "Size" present (I don't like it being present, I would have to add another condition to hide it). It would also leave the volume creation button present, which would require another condition to disable it. So I think it's simpler to hide the whole part about the volume (including the header and action buttons)

Reply:
From my understanding it's same for when the pool is active but there are not storage volumes. We still show the table headers. Therefore I would prefer to stick to that design, but I said I don't have strong feelings about it.